File: Leetcode/Python/2579. Count Total Number of Colored Cells/solution.py
# https://leetcode.com/problems/count-total-number-of-colored-cells/description

class Solution:
    def coloredCells(self, n: int) -> int:
        blank_corner = (n - 1) * n // 2
        return (n * 2 - 1) ** 2 - 4 * (blank_corner)

File: Leetcode/Typescript/2579. Count Total Number of Colored Cells/solution.ts
// https://leetcode.com/problems/count-total-number-of-colored-cells/description

function coloredCells(n: number): number {
  const blankCorner = ((n - 1) * n) / 2;
  return (2 * n - 1) ** 2 - 4 * blankCorner;
}
